src/main/java/webApp/webController.java

```diff
@@ -33,6 +34,8 @@ public Response createSurvey(@RequestBody SurveyMessage surveyMessage) {
         for (QuestionMessage question : surveyMessage.getQuestions()) {
             if (question.getType().equals("openEnded")) {
```

Comment:
I know this wasn't your change, Dharina, but @MichaelDysart we should use an enum instead of doing string equality for question types

Reply:
@TaylorAbraham 
I don't think it can be an enum.
This has to be data transmitted from the GUI.
The incoming data has to be a string in the JSON that is converted to a datatype in java.
Unless it can do some automatic conversion to enum it won't work.